src/EditorFeatures/CSharpTest/EditAndContinue/LineEditTests.cs

```diff
@@ -1417,6 +1417,50 @@ class C
                 Array.Empty<string>());
         }
 
+        [Fact]
+        public void Event_ExpressionBodyToBlock()
+        {
```

Comment:
These tests shouldn't be in "LineEditTests" class. The tests here validate that when the body is not changed, just moved to a different line we calculate line deltas correctly.